src/PhpParser/NodeFinder/LocalMethodCallFinder.php

```diff
@@ -38,15 +39,17 @@ public function match(Class_ $class, ClassMethod $classMethod): array
         $matchingMethodCalls = $this->betterNodeFinder->find(
             $class->getMethods(),
             function (Node $subNode) use ($className, $classMethodName): bool {
-                if (! $subNode instanceof MethodCall) {
+                if (! $subNode instanceof MethodCall && ! $subNode instanceof StaticCall) {
                     return false;
                 }
 
                 if (! $this->nodeNameResolver->isName($subNode->name, $classMethodName)) {
                     return false;
                 }
 
-                $callerType = $this->nodeTypeResolver->getType($subNode->var);
+                $callerType = $subNode instanceof MethodCall
```

Comment:
Would also work for more CallLike classes, like FuncCall or New_ ?

Reply:
Do you mean pass new object like this 
https://getrector.com/demo/52fee65b-774f-4c20-81f4-c2c75d123bbe